### ISSUE TITLE
Cannot find module 'simple-svelte-autocomplete' with 2.2.1 and Meteor

### ISSUE THREAD

**Issue by Prinzhorn** [pstanoev/simple-svelte-autocomplete#104]
Bundling is hard :sob: 

2.2.1 broke Meteor compatibility, probably something here https://github.com/pstanoev/simple-svelte-autocomplete/commit/38f34893879dc7d26ae727b77176ba3991cb7e21#diff-6814bf77564b4f1c92f5861e184e28fe217c080a047fefa8b73a728f755ec45c

With 2.1.0 this module was kind of working, but broke when unmounting (https://github.com/sveltejs/svelte/issues/3165). But that seems to be an issue on my (Meteor's) end, because of https://github.com/meteor/meteor/issues/11030 (I end up with two Svelte instances)

Now with 2.2.1 this module doesn't work at all and I don't know who's at fault? The steps I've provided here https://github.com/meteor/meteor/issues/11030#issuecomment-907160546 reproduce it.

```
modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:234 Uncaught Error: Cannot find module 'simple-svelte-autocomplete'
    at makeMissingError (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:234)
    at Module.resolve (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:249)
    at Module.moduleLink [as link] (modules.js?hash=858a75f4e9c6105c842d8a28c7704e50a23eb465:303)
    at app.js?hash=5636eb314611ebaa5256c10409853910a67afb9b:114
    at module (app.js?hash=5636eb314611ebaa5256c10409853910a67afb9b:494)
    at fileEvaluate (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:388)
    at Module.require (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:270)
    at Module.moduleLink [as link] (modules.js?hash=858a75f4e9c6105c842d8a28c7704e50a23eb465:310)
    at module (main.js:1)
    at fileEvaluate (modules-runtime-hot.js?hash=d12281259a6cee99786429a66f76b05f13449c7f:388)
```

**Comment by pstanoev** [comment]
Ups, sorry yes, I messed up the rollup config by mistake.
Version 2.2.2 is on the way.

**Comment by pstanoev** [comment]
Fixed in 2.2.3.

**Comment by Prinzhorn** [comment]
Thanks
